metadata/ComputerVision__5__metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "changeLog": "Release v2023.04",
+  "cpu": 0,
+  "description": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "displayName": "ComputerVision",
+  "gpu": 1,
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/user-guide/computer-vision ",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON36_CV",
```

Comment:
Is this setting still useful, considering that now we have the full image specified?

Reply:
Yes, since this is enum in code and some specific env vars, nodePool etc depend on language. So treat it just as a flag